data_steward/tools/retract_data_bq.py

```diff
@@ -700,45 +574,31 @@ def to_int(val, default=None):
         return default
 
 
-def extract_pids_from_file(pid_file_name):
-    """
-    Read specified text file and return list of person_id
-
-    :param pid_file_name: name of the file
-    :return: list of int (person_id)
-    """
-    pids_to_retract = []
-    pid_file_path = os.path.join(resources.tools_path, pid_file_name)
-    with open(pid_file_path) as f:
-        for line in f:
-            pid = to_int(line.strip())
-            if pid is None:
-                logger.warning('Found invalid person_id "%s", skipping.' % line)
-            else:
-                pids_to_retract.append(pid)
-    return pids_to_retract
-
-
 if __name__ == '__main__':
     parser = argparse.ArgumentParser(formatter_class=argparse.RawDescriptionHelpFormatter)
```

Comment:
Please specify what the command does in `ArgumentParser.description` and any caveats/limitations.

Reply:
Specify the columns needed in the table. Do we need `research_id` in the table if not deidentified (yet)?

Reply:
I've added the schema and made `research_id` nullable, along with argparse description.